.circleci/main.yml

```diff
@@ -1447,19 +1447,36 @@ jobs:
     steps:
       - checkout:
           path: ~/project
+      - install-minikube
+      - install-helm
+      - install-kubectl
+      - start-minikube
       - install-poetry
       - run: #Note: --no-ansi https://github.com/python-poetry/poetry/issues/7184
           name: Install Python Dependencies with Poetry
           command: poetry --no-ansi install
-      - install-python-test-deps
-
-      # export PACH_PYTHON_ENTERPRISE_CODE=${ENT_ACT_CODE}
-      # export export PACH_PYTHON_AUTH_TOKEN=iamroot
+      - wait-for-minikube
+      - run:
+          name: Install pach
+          command: |
+            helm install pachd ../etc/helm/pachyderm --set deployTarget=LOCAL,pachd.image.tag=${CIRCLE_SHA1},proxy.service.type=NodePort
+            kubectl create namespace enterprise --dry-run=client -o yaml | kubectl apply -f -
+            helm install pachd-enterprise ../etc/helm/pachyderm --namespace enterprise --set deployTarget=LOCAL,pachd.image.tag=${CIRCLE_SHA1},proxy.service.type=NodePort,proxy.service.httpNodePort=30081,pachd.enterpriseLicenseKey=${ENT_ACT_CODE}
+      - run:
+          name: Wait for pach # need to wait before testing kubectl as pach is not visible to kubectl yet
+          command: |
+            sleep 20
+            kubectl wait -A --for=condition=ready pod -l app=pachd --timeout=5m
+            until poetry run python -c "import pachyderm_sdk; pachyderm_sdk.Client(\"$(minikube ip)\", 30080).get_version()"; do sleep 5; done
```

Comment:
Follow on work: Move this to a package command. Can be test-only but checking connection seems useful to users as well